packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -61,6 +70,16 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
     super(biconomySmartAccountConfig);
     // Review: if it's really needed to supply factory address
     this.factoryAddress = biconomySmartAccountConfig.factoryAddress ?? DEFAULT_BICONOMY_FACTORY_ADDRESS; // This would be fetched from V2
+
+    const minimalHandlerAddress =
```

Comment:
do rename here as well. it should be consistent default fallback handler - DEFAULT_FALLBACK_HANDLER